pkg/serverless/logs/logs.go

```diff
@@ -59,10 +58,6 @@ type reportLogMetrics struct {
 	initDurationMs   float64
 }
 
-type runtimeDoneItem struct {
-	status string
```

Comment:
The `status` field on the `runtimeDoneItem` struct was never being used. Since it was the only field on the struct, I just removed the whole struct from everywhere.